src/components/AssetSearch/AssetSearch.tsx

```diff
@@ -61,78 +73,129 @@ export const AssetSearch: FC<AssetSearchProps> = ({
     [history],
   )
   const handleClick = onClick ?? defaultClickHandler
-  const [searchTermAssets, setSearchTermAssets] = useState<Asset[]>([])
   const { register, watch } = useForm<{ search: string }>({
     mode: 'onChange',
     defaultValues: {
       search: '',
     },
   })
-  const searchString = watch('search')
+  const searchString = watch('search').trim()
   const searching = useMemo(() => searchString.length > 0, [searchString])
```

Comment:
Not introduced by this PR but OCD 
```suggestion
  const isSearching = useMemo(() => searchString.length > 0, [searchString])
```